src/components/NewCommentForm.tsx

```diff
@@ -1,8 +1,80 @@
-import React from 'react';
+import React, { useState } from 'react';
+import cn from 'classnames';
+
+import { CommentData } from '../types/Comment';
+
+type Props = {
+  onAddComment: (commentData: CommentData) => void;
+  isAddingComment: boolean;
+};
+
+interface FormState {
+  name: string,
+  email: string,
+  body: string,
+}
+
+interface FormError {
+  nameError: boolean,
+  emailError: boolean,
+  bodyError: boolean,
+}
+
+export const NewCommentForm: React.FC<Props> = ({
+  onAddComment,
+  isAddingComment,
+}) => {
+  const [formState, setFormState] = useState<FormState>({
+    name: '',
+    email: '',
+    body: '',
+  });
```

Comment:
It is better to create a separate state for each property. It will improve the readability